frontend/admin/src/js/components/Table/Table.tsx

```diff
@@ -54,6 +56,47 @@ const IndeterminateCheckbox: React.FC<
     </label>
   );
 };
+const IndeterminateSelectionCheckbox: React.FC<
+  React.HTMLProps<HTMLInputElement> & { indeterminate: boolean }
+> = ({ indeterminate, ...rest }) => {
+  const ref = React.useRef<HTMLInputElement>(null);
+
+  React.useEffect(() => {
+    if (ref.current) {
+      ref.current.indeterminate = indeterminate;
+    }
+  }, [ref, indeterminate]);
+
+  return <input type="checkbox" ref={ref} {...rest} />;
+};
+
+// TODO: Try to fix these types
+const HeaderCheckbox = ({ getToggleAllPageRowsSelectedProps }) => {
```

Comment:
I can't find any good way of getting the type for 'getToggleAllPageRowsSelectedProps' here and 'rows' in RowCheckbox. I'd love to hear a good solution for this.

Reply:
There are some [types that may work in @types/react-table](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/42c09f2892ae6923649f3465b0cd4951aa6aad01/types/react-table/index.d.ts#L738).

Reply:
That helped. Thanks!